### ISSUE TITLE
remove sections /edit and /register

### ISSUE THREAD

**Issue by carolineorlando** [chrisguitarguy/Front-End-Accounts#16]
I'm using this code here (https://github.com/chrisguitarguy/Front-End-Accounts#replace-default-sections) to test removing default sections (specifically, edit and register) - but the sections still appear. I'm putting this code in functions.php (of a child theme)...

Has this functionality been updated?

**Comment by nlemoine** [comment]
From the readme: 
```
add_action('plugins_loaded', function() {
    // remove the entire account edit section
    remove_action('plugins_loaded', array(Account::instance(), '_setup'));
}, 9); // somewhere before priority 10
```

You can't remove sections from a theme as `plugins_loaded` has already been fired. 

I'll probably submit a pull request to make this plugin work as a package and change the hooks so they can be available from the theme's functions.php file.

**Comment by chrisguitarguy** [comment]
> Note that you can't remove sections from a theme as plugins_loaded has already been fired.

Not entirely true:

```php
use Chrisguitarguy\FrontEndAccounts\Account;

remove_filter('frontend_accounts_registered_sections', array(Account::instance(), 'addSection'));
```

None of the stuff that actually makes the section works fires until [`template_redirect`](https://github.com/chrisguitarguy/Front-End-Accounts/blob/42d4b5c590b6d5c23a45fd01883d4fa65cf4c28e/inc/Rewrite.php#L64-L67).

**Comment by nlemoine** [comment]
You're right, I forgot about that hook. Have you considered moving the plugins_loaded hooks to after_setup_theme? 
That would give more freedom to people when using your plugin.
